src/dealDashboard/dealInfo/dealInfo.html

```diff
@@ -144,9 +144,36 @@ <h1 class="title heading gradient">Deal Information</h1>
       </etherscanlink>
     </div>
 
+    <div if.bind="deal.registrationData.terms.daoplomatRewards"
+      class="infoSection totalRepresentatives ${showMoreRewards ? 'active' : ''}"
+      click.trigger="showMoreRewards = !showMoreRewards"
+    >
+      <span class="bodySmallTextBold infoTitle">Total DAOplomat Reward</span>
+      <span>
+        <span class="percent">
+          ${deal.registrationData.terms.daoplomatRewards.percentageAmount}
```

Comment:
```suggestion
          ${deal.registrationData.terms.daoplomatRewards.percentage}
```
because `percentageAmount` sounds redundant.

or maybe `percentageDecimal`.